content/blog/2023/Dates/vvvv beginner course winter 2023/index.md

```diff
@@ -0,0 +1,47 @@
+---
+categories: "Dates"
+author: "chk"
+date: "2023-06-10"
+title: "vvvv Beginner Class Winter 23/24"
+description: "Early bird discout until July 16th!"
+where: 
+    country_name: "The NODE Institute (online)"
+    from_date: "1697472000"
+thumb: "vvvvbeginnerwinter23-thumb.png"
+---
+
+{{< youtube "9ohSu4-vI5M" >}}
+
+Hello vvvvorld!
+
+We are happy to announce the dates for the next beginner class at The NODE Institute in the upcoming winter semester! This live class is designed as a weekly add-on to your studies or work, fully remote and scheduled every Monday evening starting on October 16th. If all you ever wanted is to learn vvvv from scratch, connect with other patching enthusiasts and benefit from some project support to kickstart your ideas, then this course is for you.
+
+Super important to mention: **Early Birds take advantage of a 20% discount until July 16th.** 
+
+Sign up here:
+https://thenodeinstitute.org/courses/ws23-vvvv-beginner-class/
+
+## What will I learn?
+
+The course covers all you need to know to start your vvvv career and entering specific intermediate or advanced topics. In 12 sessions you will learn the basics of programming in vvvv and creating real time renderings, audio and data visualizations, creative coding algorithms, best practices in structuring your applications, building interfaces and object-oriented patching in VL.
```

Comment:
"...to start your vvvv career and entering..." da stolpert man drüber. denke das wäre eher "...to start your vvvv career and enter..."

Reply:
und "...building interfaces..." maybe better "...building user interfaces..."?